.azure/windows-stack.yml

```diff
@@ -38,9 +38,11 @@ jobs:
   - bash: |
       curl -sSkL http://www.stackage.org/stack/windows-x86_64 -o /usr/bin/stack.zip
       unzip -o /usr/bin/stack.zip -d /usr/bin/
-      mkdir -p "$STACK_ROOT"
+      mkdir -p "$(cygpath $STACK_ROOT)"
```

Comment:
@pepeiborra was `cygpath` needed? afaik `mkdir -p "C:\x"` works (at least in my local msys console)

```bash
atrey@FNTSY MINGW64 /d/dev
$ mkdir -p "D:\dev\test"

atrey@FNTSY MINGW64 /d/dev
$ ls test/
```

Reply:
I don't have a Windows environment, only added it for consistency

Reply:
Fair enough but i am afraid that the existing `cygpath` was the exception (to be removed) :-/
I can remove them in a follow up if you prefer

Reply:
If you could fix the caching for the Windows CI I would be eternally grateful. Otherwise I might have to disable since the network connection to the stackage infrastructure (`casa.fpcomplete.com`) has become very unreliable and stack builds fail all the time